app/constants.py

```diff
@@ -267,4 +267,24 @@
     "team_member_ID": "1240001",
     "function_ID": "ABCDE",
     
-}
\ No newline at end of file
+}
+
+PROCESS_COMPLETION_MAIL = ('''
```

Comment:
@Couzy21 You need to provide a template that aligns to the Dowell brand. Logo, Copyright info etc. So that it is used here